### PR TITLE
Decode a Morse code message

### PR DESCRIPTION
**Pull request by Mutai-Gilbert**
Hello Reviewer 👋

In this pull request we were able to:
- Run a program using the command line.
- Use Ruby syntax for basic programming operations.
- Use Ruby to solve basic coding challenges.

### CODE REVIEW

**Comment by nahnahdev** [comment]
## Project Status: INVALIDATED :red_circle: 
Hi @Mutai-Gilbert and @calebchris000 , 

While you made a great effort in this project, I can't go ahead and review your code.


## Invalid Code Review Request

- This is because you have submitted a project, that is missing the `rubocop` linter checks [Ruby](https://github.com/microverseinc/linters-config/tree/master/ruby) in your project :100: 
- Your Current checks 
image

- What you should have
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: morse_decode.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    '....' => 'H', '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N',
+    '---' => 'O', '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T', '..-' => 'U',
+    '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y', '--..' => 'Z', '.----.' => "'", ' ' => '',
+    '   ' => ' '
+  }
+  decoded_message = morse.split(/(\s{1,})/).map { |char| morse_alphabet[char] }.join
+  print decoded_message
+end
+
+decode('.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...')
```
Comment (TedLivist) [inline morse_decode.rb]: - [ ] Based on the requirements you should have 3 methods:

  - The first one decodes one character.
  - The second one decodes a word using the first method
  - The third method decodes a sentence using the second method.

  Kindly split the implementation into three as outlined above.

- [ ] The methods should return the decoded message, unlike printing it as you have done on 10 here

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Mutai-Gilbert and @calebchris000,

Good job so far!
✔️ Descriptive README
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,2 +1,125 @@
...
+
+I would like to thank the Microverse Team! 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```
Comment (1GORDON) [inline README.md:121]: Great job team 🤝

- [ ] [Optional] Kindly fix the broken MIT license link to make your readme file more professional.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @Mutai-Gilbert & @calebchris000 ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**